benchmarks/testminer.py

```diff
@@ -25,6 +29,17 @@
     DEVNULL = open(os.devnull, 'wb')
 
 
+def get_tester(testjob):
+    type_name = testjob.testrunnerclass
+    baseurl = testjob.testrunnerurl
+    host = urlparse.urlsplit(testjob.testrunnerurl).netloc
+    (username, password) = settings.CREDENTIALS[host]
```

Comment:
I guess with this approach we can't avoid dependency from testminer to django settings. This isn't perfect and I wanted to keep testminer not depending on any django stuff. Can this be changed somehow?